pkg/test/framework/components/echo/config.go

```diff
@@ -72,6 +72,18 @@ type Config struct {
 	// ReadinessTimeout specifies the timeout that we wait the application to
 	// become ready.
 	ReadinessTimeout time.Duration
+
+	// Workloads contains the list of workloads config belonging to this echo service instance.
+	Workloads []WorkloadConfig
+}
+
+// WorkloadConfig is the config for a group of workloads (e.g. Kubernetes deployment).
+type WorkloadConfig struct {
+	Version string
```

Comment:
why do we need both a name and a version? There is no concept of a workload name currently, is there?

Reply:
There's not. But i haven't got a chance to audit the usage of the version to ensure if i can just use the version as part of deployment name without impacting existing test assumptions.

i'll take a look and refactor if possible

Reply:
addressed.